### ISSUE TITLE
Slack community link is not working

### ISSUE THREAD

**Issue by maushish** [meshery/meshery.io#1062]
### Description
The slack community link is not working anywhere be it on any repository or any link on meshery website.

### Expected Behavior
The link should work
### Screenshots
here is a screenshot 
![image](https://user-images.githubusercontent.com/114429859/223605624-51ce9f22-00ef-4a06-b123-eafbba0742a0.png)

### Enviroment:
 - OS: Mac Linux Windows
 - Browser: Chrome Safari Firefox
 - Version: 
 - Device: Desktop Mobile

---
## Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)

The meshery.io website uses Jekyll and GitHub Pages. Site content is found under the [`master` branch](https://github.com/meshery/meshery.io/tree/master).
- 📚 See the [Contributing to Meshery.io Website](https://github.com/meshery/meshery.io#contributing-to-the-mesheryio-website) section of the readme.md and other [contributing instructions](https://docs.meshery.io/project/contributing), too.
- 🎨 Wireframes and designs for Meshery UI in [Figma](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](https://slack.meshery.io)

**Comment by maushish** [comment]
Assign me this issue

**Comment by leecalcote** [comment]
"Assign this issue to me" is what I think that you mean. With a side of "please" would be a nice addition as well, @maushish.

**Comment by maushish** [comment]
Yeah my bad for being impolite let me rephrase my words -Could you please assign this issue to me @leecalcote
